spec/owner_content_resource_spec.rb

```diff
@@ -5,22 +5,6 @@
 
   include CandlepinMethods
 
-  before(:each) do
-    @owner = create_owner random_string('test_owner')
-    @user = user_client(@owner, random_string('test-user'))
-
-    @content = @cp.create_content(
-      @owner['key'], "cname", 'test-content', random_string("clabel"), "ctype", "cvendor", {}, true
-    )
-
-    @content_id = @content['id']
-
-    @product = create_product('test-product', 'some product', {:multiplier => 4})
-    @cp.add_content_to_product(@owner['key'], @product['id'], @content_id)
-  end
-
-
-
   it 'lists content in pages' do
```

Comment:
Now that you've ported these tests this file can be deleted

Reply:
Done